src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -109,12 +110,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return $this->display($input, $output, array($this->validate($twig, $template, uniqid('sf_'))));
         }
 
+        $filesInfo = $this->getFilesInfo($twig, $filename);
+
+        return $this->display($input, $output, $filesInfo);
+    }
+
+    private function getFilesInfo(\Twig_Environment $twig, $filename)
+    {
         $filesInfo = array();
-        foreach ($this->findFiles($filename) as $file) {
-            $filesInfo[] = $this->validate($twig, file_get_contents($file), $file);
+
+        if (is_array($filename)) {
```

Comment:
@stof, check recursive call done in line 124 (where 2nd argument is not an array).